### DISCUSSION TITLE
SQL Delete with multiple where

### DISCUSSION THREAD

**Discussion by medeiroshudson** [prisma/prisma#13375] [Q&A]
Im trying to perform an delete with two AND conditions, but any of theses ways work.
What am i doing wrong?

`function unrelateCostCenterXUser(costCenterId, userId) {
  return db.UserXCostCenter.delete({
    where: {
      costCenterId, userId
    }
  });
}`

`function unrelateCostCenterXUser(costCenterId, userId) {
  return db.UserXCostCenter.delete({
    where: {
      costCenterId,
      AND: [userId]
    }
  });
}`

`function unrelateCostCenterXUser(costCenterId, userId) {
  return db.UserXCostCenter.delete({
    where: {
      AND: [costCenterId, userId]
    }
  });
}`

`function unrelateCostCenterXUser(costCenterId, userId) {
  return db.UserXCostCenter.delete({
    where: {
      AND: [{ userId }, { costCenterId }]
    }
  });
}`

**Comment by nurul3101** [comment]
Hey @medeiroshudson 👋 

Are you getting any errors? Are your records not being deleted?

**Reply by vladi-stevanovic** [reply]
👋 Hello @medeiroshudson  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
